src/sentry/sentry_metrics/consumers/last_seen_updater.py

```diff
@@ -120,7 +121,21 @@ def __init__(
         self.__prefilter = LastSeenUpdaterMessageFilter(metrics=self.__metrics)
 
     def __should_accept(self, message: Message[KafkaPayload]) -> bool:
-        return not self.__prefilter.should_drop(message)
+        should_accept = not self.__prefilter.should_drop(message)
+
+        parsed_message = json.loads(message.payload.value.decode("utf-8"), use_rapid_json=True)
+        org_id = parsed_message["org_id"]
+
+        if not should_accept:
+            if org_id in settings.LAST_SEEN_UPDATER_REINDEXED_ORGS:
+                self.__metrics.incr(
+                    "last_seen_updater.reindexed_orgs.dropped_messages",
+                    tags={"organization": org_id},
+                )
```

Comment:
I may be wrong, but there was a time when certain high cardinality tags were being cut off. I do not know the exact semantics of how that was done. But in case we find that the tag is not visible it might be due to this tag being considered high cardinality.

In general though this code is ok, since the metric is only logged for a certain fixed set of organizations.

Reply:
Got it, thanks for letting me know